wurst/systems/modes/GameModeInit.wurst

```diff
@@ -0,0 +1,335 @@
+constant ENABLE_BOATS           = asList("enable-boats"          , "eb"   , "Fun"    , "Enables construction of transport boats")
+constant RAISE_ITEM_LIMIT       = asList("raise-item-limit"      , "ril"  , "Fun"    , "Increase item spawn limits")
+constant NO_INVENTORY_LIMIT     = asList("no-inventory-limit"    , "nil"  , "Fun"    , "Remove inventory limits on axes/spells/armor")
+constant FORCED_DUEL            = asList("forced-duel"           , "fd"   , "Pro"    , "Set a forced duel after a time limit, defaults to 1 h, eg: \"-fd 60\"")
+constant ELIMINATION            = asList("elimination"           , "el"   , "Pro"    , "Elimination mode (no spirit wards)")
+constant NOELIMINATION          = asList("noelimination"         , "noel" , "Pro"    , "Removed Elimination mode (spirit wards allowed)")
+constant STANDARD_1V1           = asList("standard-1v1"          , "1v1"  , "Pro"    , "Enables all standard modes for 1v1 (-sf, -rs, -fd 35)")
+constant TOURNAMENT             = asList("tournament"            , "to"   , "Pro"    , "Tournament Modes (-sf, -el, -fd)")
+public constant TEAM_GOLD       = asList("team-gold"             , "tg"   , "Pro"    , "Players on a team share gold in the same pool")
+constant NO_TRADE               = asList("no-trade"              , "nt"   , "Hard"   , "Disable trade ships and merchants")
+constant SURVIVAL               = asList("survival"              , "sv"   , "Hard"   , "This mode combines famine, cold, and slow mode")
+constant PANIC                  = asList("panic"                 , "pn"   , "Hard"   , "Stats degrade faster at 2.3 s vs 3 s")
+constant DOUBLE                 = asList("double"                , "db"   , "Hard"   , "Stats degrade twice as fast")
+constant FAMINE                 = asList("famine"                , "fm"   , "Hard"   , "Animals spawn less")
+constant FAMINE_X2              = asList("famine-x2"             , "fm2"  , "Hard"   , "More intense famine")
+constant SLOW                   = asList("slow"                  , "sl"   , "Hard"   , "Spawn less items")
+constant LOWER_FOOD_LIMIT       = asList("lower-food-limit"      , "lfl"  , "Hard"   , "Lower spawn rate for animals on the map")
+constant LOWER_ITEM_LIMIT       = asList("lower-item-limit"      , "lil"  , "Hard"   , "Lower item spawn rate on the map")
+constant TEST_MODE              = asList("test-mode"             , "tm"   , "Testing", "No one can lose, enables test commands, type -spawn or -create to get explanation")
+constant OLD_ELEMENTALIST       = asList("old-elementalist"      , "oe"   , "Testing", "Replaces new elementalist with the old elementalist.")
+constant DEV                    = asList("dev"                   , "d"    , "Testing", "Development mode (-qp, -tm, -st)")
+constant TAMEABLE_FAWN          = asList("tameable-fawn"         , "tf"   , "Testing", "Tame fawns with acorns")
```

Comment:
Let's remove this code here then I think we're good to go! Thanks for your work on this @Jaccouille